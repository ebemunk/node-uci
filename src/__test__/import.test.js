const { Engine, EngineChain } = require('../../lib')

it('exports', () => {
  expect(Engine).toMatchSnapshot()
  expect(EngineChain).toMatchSnapshot()
})
