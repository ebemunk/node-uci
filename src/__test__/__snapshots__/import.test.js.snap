// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports 1`] = `[Function]`;

exports[`exports 2`] = `[Function]`;
